src/lib/utils.js

```diff
@@ -1,4 +1,7 @@
 const transformUserInput = (item) =>
-  item.replace(/[\W_]/g, '').toLowerCase().trim();
+  item
+    .replace(/[^a-zA-Z ]/g, '')
+    .replace(/\s\s+/g, ' ')
+    .toLowerCase();
 
```

Comment:
What did you change here?

Reply:
Hi @faysvas . because you said there is a "Bug: Spaces between multi-word items disappear"?

Reply:
Hi @faysvas , when I try to use the regex we had before, it works perfectly on my side. Not sure what was the problem. 

![image](https://user-images.githubusercontent.com/37863089/118163963-dcba0b80-b422-11eb-99c3-8320d875d74e.png)

Reply:
So, this PR also closes the new issue made opened by Golfo a few days ago, great 👍🏽